Wrappers/Python/ccpi/viewer/CILViewer2D.py

```diff
@@ -1353,14 +1369,12 @@ def __init__(self, dimx=600,dimy=600, ren=None, renWin=None,iren=None, debug=Tru
         self.imageTracer.GetGlyphSource().SetRotationAngle(45.0)
         self.imageTracer.GetGlyphSource().Modified()
         self.imageTracer.ProjectToPlaneOn()
-        # Set key press activation on
-        self.imageTracer.KeyPressActivationOn()
-        # Use 't' to activate image tracer
-        self.imageTracer.SetKeyPressActivationValue('t')
+        self.imageTracer.SetHandleLeftMouseButton(True)
         
         # Set autoclose to on
         self.imageTracer.AutoCloseOn()
 
+        self.imageTracer.AddObserver(vtk.vtkWidgetEvent.Select, self.style.OnTracerModifiedEvent, 1.0)
```

Comment:
Added an observer for selecting the image tracer.
On left mouse press and middle mouse button, nothing happens
On right key press and hold we get `Tracer modified InteractionEvent` (this is what we should be getting with left and middle mouse too)
On "t" key press we get either:
Tracer modified DisableEvent
Tracer modified ModifiedEvent
or
Tracer modified ModifiedEvent
Tracer modified EnableEvent